scripts/update-models-cache.sh

```
@@ -9,27 +9,35 @@ set -euo pipefail
 
 CACHE_FILE="src/models-dev-cache.json"
 URL="https://models.dev/api.json"
+TEMP_FILE=$(mktemp)
+
+trap "rm -f '$TEMP_FILE'" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)